homeassistant/components/launch_library/sensor.py

```diff
@@ -69,15 +73,11 @@ def native_value(self) -> str | None:
     @property
     def extra_state_attributes(self) -> dict[str, Any]:
         """Return the attributes of the sensor."""
-        next_launch = self.get_next_launch()
-        return (
-            {
-                ATTR_LAUNCH_TIME: next_launch.net,
-                ATTR_AGENCY: next_launch.launch_service_provider.name,
-                ATTR_AGENCY_COUNTRY_CODE: next_launch.pad.location.country_code,
-                ATTR_STREAM: next_launch.webcast_live,
-                ATTR_ATTRIBUTION: ATTRIBUTION,
-            }
-            if next_launch
-            else {}
-        )
+        if not (next_launch := self.get_next_launch()):
+            return {}
```

Comment:
One small thing, it should return `None` when there are no attributes (please note the return type of this method needs to be adjusted to `dict[str, Any] | None`.

```suggestion
            return None
```